heron/cli/src/python/main.py

```diff
@@ -53,35 +54,39 @@ def __call__(self, parser, namespace, values, option_string=None):
     for subparsers_action in subparsers_actions:
       # get all subparsers and print help
       for choice, subparser in subparsers_action.choices.items():
-        print("Subparser '{}'".format(choice))
-        print(subparser.format_help())
+        print "Subparser '{}'".format(choice)
+        print subparser.format_help()
         return
 
 class SubcommandHelpFormatter(argparse.RawDescriptionHelpFormatter):
+  ''' SubcommandHelpFormatter '''
   def _format_action(self, action):
+    # FIXME: pylint complains about this call, need someone to verify if this is actually a problem
+    # FIXME:   Bad first argument 'RawDescriptionHelpFormatter' given to super() (bad-super-call)
```

Comment:
Seems its not easy to customize the output format. If you can make it work with the `RawDescriptionHelpFormatter`, then this would not be needed. Otherwise, its okay to have it.